sources/model/GGRealTimeMontior.m

```diff
@@ -792,9 +804,11 @@ - (void) socketIODidConnect:(SocketIOClient *)socketIO {
     if (self.socketIOConnectedBlock) {
         
         NSLog(@"\t\thandling connect success");
-        
         self.socketIOConnectedBlock(YES, nil);
         self.socketIOConnectedBlock = nil;
+        NSMutableDictionary *params = [NSMutableDictionary dictionaryWithObjectsAndKeys:
+                                       self.developerToken, @"developer_access_token",
```

Comment:
here its developer_access_token and in the above its dev_token. they both are in the websocket afaik so don't they need to be the same?